docs/index.md

```diff
@@ -25,7 +25,7 @@ From within an empty folder, add a minimal HTML file called `index.html`:
     ContainerJS status: <span id="status">initialising</span>
   </div>
 
-  <script src="https://unpkg.com/containerjs-api-bundle@0.0.2/build/containerjs-bundle.js"></script>
+  <script src="/scripts/containerjs-api.js"></script>
```

Comment:
Yeah, I'd just do a release. As we are in 0.0.x-land at the moment, just push a release out whenever you like!